test/functional/outputs/cloudwatch/forward_to_cloudwatch_test.go

```diff
@@ -203,58 +198,6 @@ var _ = Describe("[Functional][Outputs][CloudWatch] Forward Output to CloudWatch
 		})
 	})
 
-	Context("When sending infrastructure log messages to CloudWatch", func() {
```

Comment:
Why remove infra completely?

Reply:
I interpreted this as a test to ensure infra logs get to the infra tenant. The functional framework is not capable of supporting this test at this time because vector only collects logs from the node where it runs; we are unable to mock it.  Since this test was written we changed the wiring from source to sink.  The implication that I see is if logs are getting mixed in tenants, then it should be prevalent with any receiver and we have a multi-output tests